app/grandchallenge/challenges/urls.py

```diff
@@ -10,13 +10,19 @@
     ChallengeRequestList,
     ChallengeRequestStatusUpdate,
     UsersChallengeList,
+    UsersChallengeRequestList,
 )
 
 app_name = "challenges"
 
 urlpatterns = [
     path("", ChallengeList.as_view(), name="list"),
     path("my-challenges/", UsersChallengeList.as_view(), name="users-list"),
+    path(
```

Comment:
Why do we need two views? Why not one view and then filter and separate any actions based on the users permission?

Reply:
Good point actually. I thought they were too different, but that's actually not true! I'll combine them then